app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -4,62 +4,73 @@ import axios from 'axios';
 import PropTypes from 'prop-types';
 import ActiveOrders from './ActiveOrders';
 import PastOrders from './PastOrders';
+import { withRouter } from 'react-router-dom';
 import AddForm from './addForm/AddForm';
 import Tabs from '../tabs/Tabs';
 import Tab from '../tabs/Tab';
-import {
-  fetchInpatientCareSetting,
-  fetchOutpatientCareSetting,
-} from '../../actions/careSetting';
-import SearchDrug from '../searchDrug';
+import { getPastOrders } from '../../actions/pastOrders';
 
 export class SearchAndAddOrder extends React.Component {
   constructor(props) {
-    super();
+    super(props);
     this.state = {
       careSetting: 'OutPatient',
+      uuid: '',
```

Comment:
Kindly make the uuid more descriptive because we are all using `uuid`s.

Reply:
Fixed